CHANGELOG.md

```diff
@@ -1,3 +1,77 @@
+# 94.2.0
+
+This minor release introduces new APIs to MDCAppBarViewController and MDCSlider. Additionally, it includes some fixes for Banner constraints.
+
+## New Features
+
+### AppBar
+
+`MDCAppBarViewController` allows automatically adjust the height of the app bar based on its `headerStackView`.
+
+**Objective-C**
+
+```objc
+MDCAppBarViewController *appBar= [[MDCAppBarViewController alloc] init];
+appBar.shouldAdjustHeightBasedOnHeaderStackView = YES;
+```
+
+**Swift**
+
+```swift
+let appBar = MDCAppBarViewController()
+appBar.shouldAdjustHeightBasedOnHeaderStackView = true
+```
+
+### Slider 
+
+`MDCSlider` allows snapping the thumb to the nearest discrete value when the user drags the thumb.
```

Comment:
The fix/feature is actually the opposite. It's now possible to have a Continuous Slider that shows track tick marks. 
```
slider.discrete = NO;
slider.numberOfDiscreteValues = 5;
```

The reason there's a hotfix for this behavior is because the previous changes attempting to enable this behavior resulted in breaking event reporting and haptics.

Reply:
Addressed in eb980f1495c026c65f1bf24607e219f5b64c0502.

Reply:
Thanks! Looks much better. 👍